rb/lib/selenium/logger/logger_matchers.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+require 'selenium/webdriver'
+
+LEVELS = {errored: 'ERROR',
+          warned: 'WARN',
+          waring: 'WARN',
+          deprecated: 'WARN',
+          info: 'INFO',
+          informed: 'INFO',
+          debugged: 'DEBUG'}.freeze
+
+LEVELS.each_key do |level|
+  RSpec::Matchers.define :"have_#{level}" do |expected_ids, logger = nil|
```

Comment:
This will be loaded in runtime where RSpec might be unavailable. It would be better to keep it in `rb/spec/` so it's not part of the gem itself.